packages/dev/loaders/src/SPLAT/splatFileLoader.ts

```diff
@@ -45,6 +47,147 @@ export class SPLATFileLoader implements ISceneLoaderPluginAsync, ISceneLoaderPlu
         return false;
     }
 
+    /**
+     * Code from https://github.com/dylanebert/gsplat.js/blob/main/src/loaders/PLYLoader.ts

```

Comment:
We need to provide license information, if this code is copied from a different repository. "Under MIT license" or something similar.